tensionmap.py

```diff
@@ -43,6 +44,23 @@
 
 tm_update_modes = ["OBJECT", "WEIGHT_PAINT", "VERTEX_PAINT"]
 
+class Object_Data(bpy.types.Object):
```

Comment:
I think the conversion for Classes would be `ObjectData`.

I don't really like the name though. It doesn't really say what it does.

Reply:
I'm open for suggestions!

Reply:
Sorry, I was supposed to write a suggestion, then I got carried away in my review xD

`TmCachedData` ?

Also does it have to inherit the `bpy.types.Object` ? Real question here.

Reply:
I had 2 reactions:
- Wait, this is how Python does inheritance?
- Why on earth did I write this? 😄 Probably mixed it up with C++ Constructors. The __init__ function suffices.
I won't fix it for now, I'll redo this block when implementing the caching you mentioned.